content/en/logs/guide/best-practices-for-log-management.md

```diff
@@ -0,0 +1,210 @@
+
+Create an anomaly detection monitor to alert on any unexpected log indexing spikes:
+
+1. Navigate to [Monitors > New Monitor][13] and select **Anomaly**.
+2. In the **Define the metric** section, select the `datadog.estimated_usage.logs.ingested_events` metric.
+3. In the **from** field, add the `datadog_is_excluded:false` tag to monitor indexed logs and not ingested ones.
+4. In the **sum by** field, add the `service` and `datadog_index` tags, so that you are notified if a specific service spikes or stops sending logs in any index.
+5. Set the alert conditions to match your use case. For example, set the monitor to alert if the evaluated values are outside of an expected range.
+6. Add a title for the notification and a message with actionable instructions. For example, this is a notification with contextual links:
+    ```text
+    An unexpected amount of logs has been indexed in the index: {{datadog_index.name}}
+
+    1. [Check Log patterns for this service](https://app.datadoghq.com/logs/patterns?from_ts=1582549794112&live=true&to_ts=1582550694112&query=service%3A{{service.name}})
+    2. [Add an exclusion filter on the noisy pattern](https://app.datadoghq.com/logs/pipelines/indexes)
+    ``` 
+7. Click **Create**.
+
+### Alert when an indexed log volume passes a specified threshold
+
+Set up a monitor to alert if an indexed log volume in any scope of your infrastructure (for example, `service`, `availability-zone`, and so forth) is growing unexpectedly.
+
+1. Navigate to the [Log Explorer][14].
+2. Enter a [search query][15] that includes the index name (for example, `index:main`) to capture the log volume you want to monitor.
+3. Click the down arrow next to **Download as CSV** and select **Create monitor**.
+4. Add tags (for example, `host, `services, and so on) to the **group by** field.
+5. Enter the **Alert threshold** for your use case. Optionally, enter a **Warning threshold**.
+6. Add a notification title, for example: 
+    ```
+    Unexpected spike on indexed logs for service {{service.name}}
+    ```
+6.  Add a message, for example:
+    ```
+    The volume on this service exceeded the threshold. Define an additional exclusion filter or increase the sampling rate to reduce the volume.
+    ```
+7. Click **Create**.
+
+#### Alert on indexes reaching their daily quota
+
+[Set up a daily quota][16] on indexes to prevent indexing more than a given number of logs per day. If an index has a daily quota, Datadog recommends that you set the [monitor that notifies on that index's volume](#alert-when-an-indexed-log-volume-passes-a-specified-threshold) to alert when 80% of this quota is reached within the past 24 hours.
+
+An event is generated when the daily quota is reached. By default, these events have the `datadog_index` tag with the index name. You can, optionally, [create a facet][17] on the `datadog_index` tag, so that you can use `datadog_index` in the `group by` step for setting up a multi-alert monitor.
```

Comment:
```suggestion
An event is generated when the daily quota is reached. By default, these events have the `datadog_index` tag with the index name. Therefore, when this event has been generated, you can [create a facet][17] on the `datadog_index` tag, so that you can use `datadog_index` in the `group by` step for setting up a multi-alert monitor. 
```